editor/groups_editor.cpp

```diff
@@ -297,11 +298,15 @@ void GroupDialog::_load_groups(Node *p_current) {
 	}
 }
 
-void GroupDialog::_delete_group_pressed(Object *p_item, int p_column, int p_id) {
+void GroupDialog::_modify_group_pressed(Object *p_item, int p_column, int p_id) {
 	TreeItem *ti = Object::cast_to<TreeItem>(p_item);
 	if (!ti) {
 		return;
 	}
+	if (p_id == 1) { // 1 represents copy button
```

Comment:
The IDs should be made into an enum, and this should become a `switch (p_id)`.
So there's no magic number and no breakage if anyone changes the IDs.